bundles/org.openhab.binding.hue/src/main/java/org/openhab/binding/hue/internal/discovery/HueBridgeDiscoveryParticipant.java

```diff
@@ -51,13 +51,14 @@
 
     @Override
     public @Nullable DiscoveryResult createResult(RemoteDevice device) {
+        String serialNumber = device.getDetails().getSerialNumber();
         ThingUID uid = getThingUID(device);
-        if (uid != null) {
+        if (uid != null && serialNumber != null && !serialNumber.isBlank()) {
```

Comment:
Why shouldn't host, port and protocol be set, just because no serial number is available?

Reply:
Because the serial number is required to build the thing UID and is the representation property of the thinj.

Reply:
That is an assumption on how `getThingUID`, but that could potentially change at any time in future. Imho such implicit assumptions that are based on a current implementation should not be made.

Reply:
Ok, I will change.

Reply:
Done